packages/features/bookings/Booker/components/OverlayCalendar/OverlayCalendarSettingsModal.tsx

```diff
@@ -93,8 +95,12 @@ export function OverlayCalendarSettingsModal({
                                       "h-10 w-10",
                                       item.integration.logo.includes("-dark") && "dark:invert"
                                     )}
-                                    src={item.integration.logo}
-                                    alt={item.integration.title}
+                                    src={
```

Comment:
We don't store images in platform thats why for atoms the image fetches comes from app.cal.com